pycyphal/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -254,6 +265,32 @@ def configure_acceptance_filters(self, configuration: typing.Sequence[FilterConf
         _logger.debug("%s: Acceptance filters activated: %s", self, ", ".join(map(str, configuration)))
         self._bus.set_filters(filters)
 
+    def transmit_thread_worker(self) -> None:
+        try:
+            while not self._closed:
+                tx_tuple = self._tx_queue.get(block=True)
+                try:
+                    if self._closed or tx_tuple is None:
+                        return
+
+                    message: can.Message = tx_tuple.msg
+                    timeout: float = tx_tuple.timeout
+                    future: asyncio.Future[None] = tx_tuple.future
+                    loop: asyncio.AbstractEventLoop = tx_tuple.loop
+                    self._bus.send(message, timeout)
+                    loop.call_soon_threadsafe(lambda: future.set_result(None))
+                except Exception as ex:
+                    if future:
+                        loop.call_soon_threadsafe(lambda: future.set_exception(ex))
+                    else:
+                        _logger.debug("Future is None, cannot set exception: %s", ex, exc_info=True)
```

Comment:
`if future` makes no sense here because `future` may be undefined, which is not the same as being None.

```suggestion
                    tx_tuple.loop.call_soon_threadsafe(lambda: tx_tuple.future.set_exception(ex))
```